### ISSUE TITLE
Can't access RequestTelemetry via controller DI

### ISSUE THREAD

**Issue by davidmatson** [microsoft/ApplicationInsights-aspnetcore#368]
The wiki shows using controller constructor DI to get a RequestTelemetry object:
https://github.com/Microsoft/ApplicationInsights-aspnetcore/wiki/Configure
under "Add properties for request data item"

It looks like that doesn't work anymore. Should it?

Two requests:
1. Update the documentation/wiki
2. Provide an easy way to do this

The only currently way I can find to do it is to use `HttpContext.Features.Get<RequestTelemetry>`.

**Comment by henkmollema** [comment]
The Wiki has been updated: https://github.com/Microsoft/ApplicationInsights-aspnetcore/wiki/Configure/_compare/fb376607898b2ee841c51d9d6a3a83cbfa347209...a6fdc4373bda55adc7900f40e44822b990a93389.

**Comment by froitag** [comment]
The Wiki was updated by me because I didn't want others to waste time figuring out injecting RequestTelemtry via DI doesn't work anymore (like I did).

I'll gladly reverse the change if this turns out as a bug and the original behavior is restored.

However, I don't think this is of any importance as long as the docs are correct. Although of course it would only be consequent accessing RequestTelemetry via DI since this is how you access TelemtryClient, or accessing both via `HttpContext.Features`.
